snap/quota/quota.go

```diff
@@ -153,6 +221,253 @@ func (grp *Group) SliceFileName() string {
 	return buf.String()
 }
 
+// groupQuotaStats contains information about current quotas of a group
+// and is used by getGroupQuotaStats to contain this information
+type groupQuotaStats struct {
```

Comment:
stats is make is sound like runtime information, Allocation? Bookkeeping?

Reply:
Really good point, I'll change it